permissions_validators/src/public_blockchain/mod.rs

```diff
@@ -64,6 +66,26 @@ impl From<PredefinedPermissionToken> for PermissionToken {
     }
 }
 
+/// Lists ids of all predefined permission tokens, e.g. for easier
+/// registration in genesis block.
+pub fn default_permission_token_definitions(
+) -> [&'static PermissionTokenDefinition; PredefinedPermissionToken::VARIANT_COUNT] {
+    [
+        unregister::CanUnregisterAssetWithDefinition::definition(),
+        burn::CanBurnAssetWithDefinition::definition(),
+        burn::CanBurnUserAssets::definition(),
+        key_value::CanSetKeyValueInUserAssets::definition(),
+        key_value::CanRemoveKeyValueInUserAssets::definition(),
+        key_value::CanSetKeyValueInUserMetadata::definition(),
+        key_value::CanRemoveKeyValueInUserMetadata::definition(),
+        key_value::CanSetKeyValueInAssetDefinition::definition(),
+        key_value::CanRemoveKeyValueInAssetDefinition::definition(),
+        mint::CanMintUserAssetDefinitions::definition(),
+        transfer::CanTransferUserAssets::definition(),
+        transfer::CanTransferOnlyFixedNumberOfTimesPerPeriod::definition(),
+    ]
+}
```

Comment:
Theoretically this can be macro-generated as well. I don't insist just an idea how to reduce the amount of work (and possible errors) when someone will be adding a new variant

Reply:
I strongly think that we should make a proc-macro that generates the information.

Reply:
However this is best left as a chore.